src/Resources/ResourceManager/SdkExtensions/ResourcesExtensions.cs

```diff
@@ -195,6 +195,38 @@ private static string ConstructTemplateLinkView(TemplateLink templateLink)
             return result.ToString();
         }
 
+        public static string ConstructOutputTable(IDictionary<string, object> dictionary)
+        {
+            if (dictionary == null)
+            {
+                return null;
+            }
+
+            var maxNameLength = 18;
+            dictionary.Keys.ForEach(k => maxNameLength = Math.Max(maxNameLength, k.Length + 2));
+
+            var maxTypeLength = 25;
+            dictionary.Values.ForEach(v => maxTypeLength = Math.Max(maxTypeLength, v.ToString().Split('\n').OrderByDescending(s => s.Length).First().Length + 2));
+
+            StringBuilder output = new StringBuilder();
+                
+                if (dictionary.Count > 0)
+                {
+                    string rowFormat = "{0, -" + maxNameLength + "}  {1, -" + maxTypeLength + "}\r\n";
+                    output.AppendLine();
+                    output.AppendFormat(rowFormat, "Key", "Value");
+                    output.AppendFormat(rowFormat, GeneralUtilities.GenerateSeparator(maxNameLength, "="), GeneralUtilities.GenerateSeparator(maxNameLength, "="));
```

Comment:
> GeneralUtilities.GenerateSeparator(maxNameLength, "=") [](start = 106, length = 55)

A bit confused with the use of `maxTypeLength`. You don't seem to use it in the AppendFormat, do we need it then?

Reply:
Ah yes, I think I can get rid of that variable. I was using it earlier to determine the length of the separating line at the top. It's redundant now.